documentation/wiki/MSBuild-Tips-&-Tricks.md

```diff
@@ -13,6 +13,9 @@ See the [MSBuild Command-Line Reference](https://docs.microsoft.com/visualstudio
    * Passes parameters to the file logger. If you want to attach multiple file loggers, you do so by specifying additional parameters in the switches /flp1, /flp2, /flp3, and so on.
 
 # Environment Variables
+
+ * `MSBUILDDEBUGENGINE=1` & `MSBUILDDEBUGPATH=<DIRECTORY>`
```

Comment:
MSBuildDebugEngine is technically pascal-cased. Why? Don't know. Windows doesn't care. I seem to recall we talked about whether we should change it to be all upper case like most of our env vars but ultimately decided that since it had shipped this way, we'd leave it.